CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Braintree Android Drop-In Release Notes
 
+## unreleased
+
+* Include `braintree-core`, `three-d-secure`, `paypal`, `venmo`, `google-pay`, `card`, `data-collector`, and `union-pay` as api
+* Add `DropInClient`
+* Add `DropInResultCallback`
+* Add `FetchMostRecentPaymentMethodCallback`
+* Add `VaultedPaymentMethodSelectedListener`
```

Comment:
Looks like we never added the doc strings for some of these new classes - do we want to do that in this PR or after beta1?